xbmc/cores/VideoPlayer/VideoRenderers/LinuxRendererGL.cpp

```diff
@@ -1662,8 +1662,10 @@ void CLinuxRendererGL::RenderRGB(int index, int field)
   glBindTexture(m_textureTarget, 0);
 }
 
-bool CLinuxRendererGL::RenderCapture(CRenderCapture* capture)
+bool CLinuxRendererGL::RenderCapture(int index, CRenderCapture* capture)
 {
+  m_iYV12RenderBuffer = index;
```

Comment:
Not sure if changing the state of renderer is a good idea. I would try to go with a local variable here instead of setting the member.

Reply:
Thank you for the hint!
Since `m_iYV12RenderBuffer` is only used once inside the function, I could directly pass the `index `to the `Render()` function, right? Then there is no need for an additional variable.

e.g. like this: `  Render(RENDER_FLAG_NOOSD, index);`

The build and test locally was successful withthis change.

Reply:
> I could directly pass the index to the Render() function, right?

Yes.

Reply:
Done and changes pushed.